### DISCUSSION TITLE
Export to data warehouses (BigQuery, Snowflake, etc.) via Blob Storage (S3, GCS, Azure Blob Storage)

### DISCUSSION THREAD

**Discussion by rohan-mehta** [langfuse/langfuse#2131] [Ideas]
### Describe the feature or potential improvement

Export to BigQuery, and other data warehouses - snowflake, redshift etc. Useful for self-serve data analysis.

**Comment by shure-dev** [comment]
https://langfuse.com/docs/analytics/posthog

**Comment by marcklingen** [comment]
Langfuse allows exports to blob storage now: https://langfuse.com/docs/query-traces#blob-storage

**Comment by farazhm** [comment]
Is there plan to add s3 support @marcklingen? easier to do a dump to snowflake from there for my company

**Reply by marcklingen** [reply]
Can you have a look here, export to S3 is available: https://langfuse.com/docs/query-traces#blob-storage
